modules/oc-by-example-content.adoc

```diff
@@ -577,7 +577,7 @@ Display merged kubeconfig settings or a specified kubeconfig file
   # Show merged kubeconfig settings
   oc config view
   
-  # Show merged kubeconfig settings and raw certificate data
+  # Show merged kubeconfig settings and raw certificate data and exposed secrets
```

Comment:
Should this be:
Show merged kubeconfig settings, raw certificate data, and exposed secrets
??